clrtrust.in

```diff
@@ -108,6 +108,30 @@ EINVAL=22       # invalid argument
 EBADST=127      # invalid state / health check does not pass
 EERR=255        # general error
 
+##### LOCATION OF THE HELPER
+# always favor env override. if not, prefer the helper that is found next to the
+# script itself (it indicates the development environment). if not, look at the
+# configured location.
+LIBEXEC_DIR=LIBEXEC_CONFIG_VALUE
+CLR_HELPER_NAME=clrtrust-helper
+if [ -n "${CLR_LIBEXEC_DIR}" ]; then # environment override
```

Comment:
This is fine. If I was writing it I would do it more like

    for d in ${CLR_LIBEXEC_DIR+"$CLR_LIB_EXEC_DIR"} "$(dirname "$0")" "${LIBEXEC_DIR}"
    do if [ -x "$d/${CLR_HELPER_NAME}" ] ; then
        CLR_HELPER_CMD="$d/${CLR_HELPER_NAME}"
        break
        fi
     done